### PR TITLE
ALFREDAPI-538 Fixed issue where errors related to jackson libraries would show up in the Alfresco logs while starting

### PR DESCRIPTION
**Pull request by kpiot123**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-538

- [ ] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/user/rest-api/index.html#rest-http-result-codes)?
- [ ] Is error handling done through a method annotated with `@ExceptionHandler` in the webscript classes?
- [ ] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [ ] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

File: apix-impl/build.gradle [hunk 1]
```diff
@@ -33,23 +37,16 @@ allprojects {
...
         alfrescoProvided "org.alfresco:alfresco-repository"
         alfrescoProvided 'org.alfresco:alfresco-remote-api'
+        alfrescoProvided 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
 
         testImplementation platform("org.alfresco:acs-community-packaging:${alfresco_version}")
         testImplementation 'org.alfresco:alfresco-repository'
         testImplementation 'org.alfresco:alfresco-remote-api'
         testImplementation 'org.alfresco:alfresco-data-model'
-        testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.25.1'
+        testImplementation 'org.mockito:mockito-core'
```
Comment (kpiot123) [inline apix-impl/build.gradle:49]: Mockito is already provided by Alfresco so we get the version from the "testImplementation platform".

File: apix-impl/build.gradle [hunk 2]
```diff
@@ -33,23 +37,16 @@ allprojects {
     apply plugin: 'java-library'
 
     dependencies {
-        api(project(":apix-interface")) {
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
-        }
-        implementation group: 'commons-lang', name: 'commons-lang', version: '1.0'
```
Comment (kpiot123) [inline apix-impl/build.gradle:40]: This dependency should not be needed anymore.

File: apix-impl/build.gradle [hunk 3]
```diff
@@ -33,23 +37,16 @@ allprojects {
     apply plugin: 'java-library'
 
     dependencies {
-        api(project(":apix-interface")) {
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
```
Comment (kpiot123) [inline apix-impl/build.gradle:38]: We don't have to exclude these dependencies because on apix-interface they are add as "compileOnly"

**Review by WimCrols — APPROVED** [review]
Thank you @kpiot123 for the fix and the clear explanation.

**Pull request merged**